Sources/SKSwiftPMWorkspace/SwiftPMWorkspace.swift

```diff
@@ -49,7 +49,7 @@ public final class SwiftPMWorkspace {
   public weak var delegate: SKCore.BuildSystemDelegate? = nil
 
   let workspacePath: AbsolutePath
-  let packageRoot: AbsolutePath
+  public let packageRoot: AbsolutePath
```

Comment:
Our general paradigm for these would be to have

```suggestion
  let packageRoot: AbsolutePath
  /// *Public for testing*
  public let _packageRoot: AbsolutePath { packageRoot }
```

That way we can check that the underscored method is only used from the tests using call hierarchy.

Reply:
done!